test/e2e/atlas/kubernetes_config_generate_test.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build e2e || (atlas && project && kubernetes)
+
+package atlas_test
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongodb-atlas-cli/test/e2e"
+	atlasV1 "github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	k8syaml "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+func getK8SEntities(data []byte) ([]runtime.Object, error) {
+	b := bufio.NewReader(bytes.NewReader(data))
+	r := k8syaml.NewYAMLReader(b)
+
+	var result []runtime.Object
+
+	for {
+		doc, err := r.Read()
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			return nil, err
+		}
+
+		d := scheme.Codecs.UniversalDeserializer()
+		obj, _, err := d.Decode(doc, nil, nil)
+		if err != nil {
+			// if document is not a K8S object, skip it
+			continue
+		}
+		if obj != nil {
+			result = append(result, obj)
+		}
+	}
+	return result, nil
+}
+
+func TestKubernetesConfigGenerate(t *testing.T) {
+	g := newAtlasE2ETestGenerator(t)
+	g.generateProjectAndCluster("importer-test")
+	g.generateCluster()
```

Comment:
I believe that `generateProjectAndCluster` already creates a project and a cluster for the e2e test
```suggestion
	g.generateProjectAndCluster("importer-test")
```